ui/desktop/app/templates/scopes/scope/projects/targets.hbs

```diff
@@ -41,7 +41,10 @@
                 <Card
                   @heading={{concat
                     (t 'resources.session.credential.title')
-                    (if credential.purpose (concat ' (' credential.purpose ')'))
+                    (if
+                      credential.library.name
+                      (concat ' (' credential.library.name ')')
```

Comment:
Name is optional, so the parens should be conditional.  You could try an inline if:

```
(if credential.library.name (concat ' (' credential.library.name ')'))
```